src/sdk/pynni/nni/compression/tensorflow/builtin_pruners.py

```diff
@@ -33,7 +33,6 @@ def calc_mask(self, layer, config):
 class AGP_Pruner(Pruner):
     """An automated gradual pruning algorithm that prunes the smallest magnitude
```

Comment:
shall we offer some test metric for user?

Reply:
We will discuss this on Monday.